requirements.txt

```diff
@@ -1,6 +1,6 @@
 # -i https://pypi.org/simple
-certifi==2018.10.15
+certifi<=2018.10.15
```

Comment:
Do we want to limit our certifi bundle to that version at the latest? I think in the integrations repo we specify a minimum version (i.e. `certifi>=2018.10.15`) so that new updates to the cert bundle are pulled automatically, since the changes to the library will not be breaking.